### ISSUE TITLE
Add partial substitution of docstrings -- expansion of .yml file support

### ISSUE THREAD

**Issue by s1113950** [gangverk/flask-swagger#39]
I was curious if there will be support for doing a partial substitution of a doc string instead of an entire replace if you specify a .yml file?

What I'm trying to do is something like:

```
def my_func():
    """
    My func title

    special text for my func

    swagger_from_file: file.yml

    more special text
    """
```
And have the `special text for my func` and `more special text` be represented as well. Currently the whole doc string gets replaced by what's in the `file.yml` file. This is especially useful when there's a bunch of endpoints with mostly common functionality but differ by a few parameters for example.

I'd be happy to make a PR for this if you see it as a worthy addition! 😄

**Comment by atlithorn** [comment]
So basically a merge of swagger found in docstrings and swagger_from_file? 
Sounds very worthy, I assume that inline swagger would take preference, right?

**Comment by s1113950** [comment]
Yep! It would be like the full doc = `text here` + {yml_file_data} + `text here`.

**Comment by s1113950** [comment]
Oh wait, you're talking about the `yml` file being like a template that can be overridden by the inline swagger? I can do that as well!

**Comment by atlithorn** [comment]
Yes - I think that would be clearer and more useful than string concatenation.

**Comment by s1113950** [comment]
Agreed!
